packages/vm/src/index.ts

```diff
@@ -163,7 +165,11 @@ export default class VM extends AsyncEventEmitter {
   protected readonly _opts: VMOpts
   protected _isInitialized: boolean = false
   protected readonly _allowUnlimitedContractSize: boolean
-  protected _opcodes: OpcodeList
+  // This opcode data is always set since `updateActiveOpcodes()` is called in the constructor
+  protected _opcodes!: OpcodeList
+  protected _handlers!: Map<number, OpHandler>
+  protected _dynamicGasHandlers!: Map<number, AsyncDynamicGasHandler | SyncDynamicGasHandler>
```

Comment:
If `updateActiveOpcodes()` is called in the constructor, are these assigments then not detected by TypeScript (and the `!` additions therefore unnecessary)? 🤔

Reply:
You would assume that TypeScript can detect this, especially because (1) the method is called in the constructor and (2) it does not have any branching (no if statements). However, it does not detect this, I had to add these `!`s. I can re-check though but I'm rather sure it threw if I removed them.